src/lib.rs

```diff
@@ -62,8 +62,8 @@ impl Unpin for Body {}
 
 impl Stream for Body {
     type Item = Result<Bytes, std::io::Error>;
-    fn poll_next(mut self: Pin<&mut Self>, waker: &Waker) -> Poll<Option<Self::Item>> {
-        Pin::new(&mut self.stream).poll_next(waker)
+    fn poll_next(mut self: Pin<&mut Self>, cx: &mut Context) -> Poll<Option<Self::Item>> {
```

Comment:
This is missing a lifetime parameter:

```suggestion
    fn poll_next(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>> {
```

Reply:
Ah, thanks for fix up! and the pragmas - so reliant on that now.